cypress/support/pages/homeCatalogue.pageObject.js

```diff
@@ -17,6 +17,89 @@ class HomeAndCataloguePageObject extends PageObject {
     cy.contains('.hrefch', product)
       .click();
   }
+
+  clickOnAddToCartBtn() {
+    cy.contains('.btn', 'Add to cart')
+      .click();
+  }
+
+  verifyAddingToCart() {
+    return this.assertAllert('Product added');
+  }
+
+  verifyProductsInCart() {
+    cy.contains('#tbodyid', 'Sony vaio i7')
+      .should('contain', 'Sony vaio i7');
```

Comment:
The assertion here is redundant. The `cy.contains` command already checks for the presence of the text, so the `should('contain', 'Sony vaio i7')` is unnecessary. Consider removing the redundant assertion.